src/assets/i18n/en.json

```diff
@@ -342,6 +342,8 @@
       "controlledStop": "Controlled stop",
       "eyesightTest": "Eyesight test",
       "uncoupleRecouple": "Uncouple / Recouple",
+      "reverseManoeuvreObservation": "Reverse Manoeuvre Observation",
```

Comment:
These changes also need to be ported into the ```assets/i18n/cy.json``` where translations needs to be confirmed